tests/test_chi_board_elections.py

```diff
@@ -0,0 +1,137 @@
+from datetime import date, time
+
+import pytest
+from tests.utils import file_response
+
+from city_scrapers.constants import COMMISSION
+from city_scrapers.spiders.chi_board_elections import ChiBoardElectionsSpider
+
+test_response = file_response('files/chi_board_elections.html')
```

Comment:
`freezegun` should be added here so that the test results don't depend on when it runs. We've got examples of this in some of the other scraper test files

Reply:
Should I run freezegun twice, since I'm looking at two pages?

Reply:
You can just call the `stop()` method after the second spider fixture is created, that way it's running for both

Reply:
Oh okay, I updated the test file